### PR TITLE
[#401] Support GHC-8.8.2

### LINKED ISSUE #401: Support GHC-8.8.2
Currently blocked by the [absence of GHC-8.8.2 support in GitHub actions](https://github.com/actions/setup-haskell/issues/6)

### PR DESCRIPTION
**Pull request by vrom911**
Resolves #401

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,14 +18,12 @@ matrix:
...
-  - ghc: 8.8.1
+  - ghc: 8.8.2
 
   # Stack, linux
 
-  - ghc: 8.6.5
+  - ghc: 8.8.2
     env: STACK_YAML="$TRAVIS_BUILD_DIR/stack.yaml" DEPLOY=yes
-  - ghc: 8.8.1
-    env: STACK_YAML="$TRAVIS_BUILD_DIR/stack-nightly.yaml" DEPLOY=no
```
Comment (chshersh) [inline .travis.yml]: Nice cleanup of this file! 👍

File: summoner-cli/test/Test/Show.hs [hunk 2]
```diff
@@ -19,5 +19,5 @@ goldenGhcTable =
   , "GHC-8.2.2     base-4.10.1.0   lts-11.22         "
   , "GHC-8.4.4     base-4.11.1.0   lts-12.26         "
   , "GHC-8.6.5     base-4.12.0.0   lts-14.18         "
-  , "GHC-8.8.1     base-4.13.0.0   nightly-2019-12-25"
+  , "GHC-8.8.2     base-4.13.0.0   lts-15.0          "
```
Comment (chshersh) [inline summoner-cli/test/Test/Show.hs]: Should we remove this padding from all values in the table? Or is it also useful for TUI?

File: summoner-cli/src/Summoner/GhcVer.hs [hunk 3]
```diff
@@ -63,7 +63,7 @@ latestLts = \case
     Ghc822  -> "lts-11.22"
     Ghc844  -> "lts-12.26"
     Ghc865  -> "lts-14.18"
```
[the remark this answers is not in the document]
Comment (vrom911) [reply summoner-cli/src/Summoner/GhcVer.hs]: Nice catch!

File: summoner-cli/test/Test/Show.hs (hunk 2 above)
Comment (vrom911) [reply summoner-cli/test/Test/Show.hs]: This is for the table builder. This pad is supposed to take into consideration the maximum possible alignment, in here I counted the nightly as well. But you are right, I can improve this :)

File: summoner-cli/src/Summoner/GhcVer.hs [hunk 4]
```diff
@@ -144,4 +144,4 @@ formatGhcMeta GhcMeta{..} =
     <> "  "
     <> T.justifyLeft 14 ' ' gmBase
     <> "  "
-    <> T.justifyLeft 18 ' ' gmResolver
```
Comment (vrom911) [inline summoner-cli/src/Summoner/GhcVer.hs]: As this padding is not used for border creation it doesn't make much sense to keep it in the first place :)

**Pull request merged**